content/en/tracing/trace_collection/dd_libraries/cpp.md

```diff
@@ -86,23 +86,29 @@ After the Agent is installed, follow these steps to add the Datadog tracing libr
 ### Compile against dd-opentracing-cpp
 
 ```bash
-# Gets the latest release version number from GitHub.
-get_latest_release() {
-  wget -qO- "https://api.github.com/repos/$1/releases/latest" |
-    grep '"tag_name":' |
-    sed -E 's/.*"([^"]+)".*/\1/';
-}
-DD_OPENTRACING_CPP_VERSION="$(get_latest_release DataDog/dd-opentracing-cpp)"
-# Download and install dd-opentracing-cpp library.
-wget https://github.com/DataDog/dd-opentracing-cpp/archive/${DD_OPENTRACING_CPP_VERSION}.tar.gz -O dd-opentracing-cpp.tar.gz
-mkdir -p dd-opentracing-cpp/.build
-tar zxvf dd-opentracing-cpp.tar.gz -C ./dd-opentracing-cpp/ --strip-components=1
-cd dd-opentracing-cpp/.build
-# Download and install the correct version of opentracing-cpp, & other deps.
-../scripts/install_dependencies.sh
-cmake ..
-make
-make install
+# Requires the "jq" command, which can be installed via
+# the package manager, for example "apt install jq",
+# "apk add jq", "yum install jq".
+if ! command -v jq >/dev/null 2>&1; then
+  >&2 echo "jq command not found. Install using the local package manager."
+else
```

Comment:
I put the entire real body of the script in an `else` branch so that I don't need to call `exit` above. If a user is copy/pasting this code into their shell, `exit` would exit the command shell, which would be rude.